python/ray/experimental/state.py

```diff
@@ -329,6 +329,9 @@ def log_files(self):
       filename = filename.decode("ascii")
       filename_components = filename.split(":")
       ip_addr = filename_components[1]
+      suffix = filename_components[2].split("/")
+      client = suffix[3].split("-")
+      client_id = client[0]
```

Comment:
I don't think these lines will produce unique client IDs (e.g., all of the redis servers seem to get collapsed into just the client ID `'redis'`).

The problem is that the client IDs I was hoping to use (the ones stored in the `DBClientID` field in `ray.global_state.client_table()`) don't even exist for all processes.

Maybe using client IDs is not the right way to do it, we should think about this more.